src/components/settings-menu/SettingsSection.js

```diff
@@ -190,10 +179,17 @@ const SettingsSection = ({
             onPressShowSecret={onPressShowSecret}
           >
             <ListItemArrowGroup>
-              {areBackedUp ? <CheckmarkIcon /> : <WarningIcon />}
+              {!areBackedUp ? <CheckmarkIcon /> : <WarningIcon />}
```

Comment:
Did you mean to do this?

Reply:
good catch, didn't mean to leave that in, my b